react/features/conference/components/native/Conference.js

```diff
@@ -214,10 +225,23 @@ class Conference extends AbstractConference<Props, State> {
      * @returns {ReactElement}
      */
     render() {
-        const { _fullscreenEnabled } = this.props;
+        const {
+            _backgroundImageUrlBranding,
+            _brandingStyles,
+            _fullscreenEnabled
+        } = this.props;
 
         return (
-            <Container style = { styles.conference }>
+            <Container
+                style = { [
+                    styles.conference,
+                    _brandingStyles.conference
+                ] }>
+                <BrandingImage
```

Comment:
This needs to work both with png and svg backgrounds. Can you pl create a wrapper component, say `BackgroundImage` which renders a `SvgUri` (or should we use `SvgCssUri` ?) or an `ImageBackground`, depending on the image type. For now we can do a naive check like if the URL ends in `svg` it's an svg.